extensions/reactive-datasource/runtime/pom.xml

```diff
@@ -26,6 +26,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-datasource</artifactId>
         </dependency>
+        <dependency>
```

Comment:
So we are adding this dependency because of the reuse of some configuration class. I just want to be sure there is not a dragon hidden under a stone.

Reply:
Yes, the dependency was added to share the ssl helper class.